govcd/org.go

```diff
@@ -32,20 +33,39 @@ func NewOrg(client *Client) *Org {
 	}
 }
 
-// AdminOrg gives an admin representation of an org.
-// Administrators can delete and update orgs with an admin org object.
-// AdminOrg includes all members of the Org element, and adds several
-// elements that can be viewed and modified only by system administrators.
-type AdminOrg struct {
-	AdminOrg *types.AdminOrg
-	c        *Client
-}
+// Given a valid catalog name, FindCatalog returns a Catalog object.
+// If no catalog is found, then returns an empty catalog and no error.
+// Otherwise it returns an error.
+func (org *Org) FindCatalog(catalogName string) (Catalog, error) {
```

Comment:
'av' is not descriptive please use a different name.

Reply:
fixed